casbin/model/policy.py

```diff
@@ -91,11 +91,6 @@ def add_policies(self,sec,ptype,rules):
     def remove_policy(self, sec, ptype, rule):
         """removes a policy rule from the model."""
 
-        if sec not in self.model.keys():
-            return False
-        if ptype not in self.model[sec]:
-            return False
-
```

Comment:
Why not do the same for `remove_policies()`?

Reply:
Good idea